lib/WsRtspVideo.tsx

```diff
@@ -122,14 +126,16 @@ export const WsRtspVideo: React.FC<WsRtspVideoProps> = ({
       })
       setPipeline(pipeline)
 
+      let scheduler: utils.Scheduler<ScheduledMessage> | undefined
       if (metadataHandler !== undefined) {
-        attachMetadataHandler(pipeline, metadataHandler)
+        scheduler = attachMetadataHandler(pipeline, metadataHandler)
       }
 
       return () => {
         debugLog('close pipeline and clear video')
         pipeline.close()
         videoEl.src = ''
+        scheduler && scheduler.reset()
```

Comment:
Cleaner to use:
`scheduler?.reset()`

Reply:
Yes, and we still don't have adapted the lint rules so I missed this.